recipe/meta.yaml

```diff
@@ -2,36 +2,66 @@
 {% set version = "1.7.14"%}
 
 {% set sha256 = "1f3c05e60882f014236a1c93fac05afde49b0fcb67aabb4df0a11b96b5aa5d24" %}
+{% set sha256_win64 = "7f36a84bf7563d762046cda5fd801614fecd12a4724a9e9002011346b1b7130d" %}
+
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/kubernetes/{{ name }}/archive/v{{ version }}.tar.gz
-  fn: {{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/kubernetes/{{ name }}/archive/v{{ version }}.tar.gz  # [not win]
+  url: https://dl.k8s.io/v{{ version }}/{{ name }}-node-windows-amd64.tar.gz  # [win]
+  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
+  sha256: {{ sha256 }}  # [not win]
+
+  fn: {{ name }}-client-{{ version }}.tar.gz  # [win]
+  sha256: {{ sha256_win64 }}  # [win]
 
 build:
-  skip: True  # [not linux]
-  number: 0
+  skip: True  # [win]
```

Comment:
Might want to skip Windows 32-bit whenever Windows is enabled if these are 64-bit packages only.

Reply:
I will take care of it in the windows PR.